cardano-node-chairman/test/Test/Cardano/Node/Chairman/Shelley.hs

```diff
@@ -57,6 +57,9 @@ import qualified System.Process as IO
 
 {- HLINT ignore "Redundant <&>" -}
 {- HLINT ignore "Redundant flip" -}
+{- HLINT ignore "Reduce duplication" -}
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
Not sure why I need to all use the old style pragma to suppress this `hlint` warning.